src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2AuctionTimer.tsx

```diff
@@ -0,0 +1,59 @@
+import { Spacer } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { AuctionTimerFragmentContainer } from "Components/AuctionTimer"
+
+import { ArtworkSidebar2AuctionTimer_artwork$data } from "__generated__/ArtworkSidebar2AuctionTimer_artwork.graphql"
+import { LotTimerFragmentContainer } from "Apps/Artwork/Components/ArtworkSidebar/LotTimer"
+import { lotIsClosed } from "Apps/Artwork/Utils/lotIsClosed"
+
+interface ArtworkSidebar2AuctionTimerProps {
+  artwork: ArtworkSidebar2AuctionTimer_artwork$data
+}
+
+export const ArtworkSidebar2AuctionTimer: React.FC<ArtworkSidebar2AuctionTimerProps> = ({
+  artwork,
+}) => {
+  const { sale, saleArtwork } = artwork
+
+  return (
+    <>
+      {sale &&
+        saleArtwork &&
+        !lotIsClosed(sale, saleArtwork) &&
+        (sale?.cascadingEndTimeIntervalMinutes ? (
+          <>
+            <Spacer mt={2} />
+            {/* @ts-ignore RELAY UPGRADE 13 */}
```

Comment:
@gkartalis - all of these ignores can be removed now!

Reply:
woops good catch! will do a follow up removing everything that I missed